### PR TITLE
Change FBC logic to not call IIB only when ocp_version >=4.13

### PR DESCRIPTION
**Pull request by yashvardhannanavati**
Stonesoup intends to start supporting indexes only from OCP v4.13

### CODE REVIEW

File: pubtools/_quay/operator_pusher.py [hunk 1]
```diff
@@ -482,25 +482,25 @@ def build_index_images(self):
...
-                    if tuple([int(x) for x in version.replace("v", "").split(".")]) > (4, 10)
+                    if tuple([int(x) for x in version.replace("v", "").split(".")]) > (4, 12)
                 ]
                 and items_opted_in[id(item)]
             ):
                 item.add_error(
                     "INVALIDFILE",
                     "Cannot push item to index image "
-                    "as it supports both <= 4.10 and >= 4.11 and is opted in FBC: {item}".format(
+                    "as it supports both <= 4.13 and >= 4.12 and is opted in FBC: {item}".format(
```
Comment (emilyzheng) [inline pubtools/_quay/operator_pusher.py:497]: Should be '<=4.12 and >=4.13' or '<4.13 and > 4.12'?

**Review by chandwanitulsi — COMMENTED** [review]
minor change, other than that looks good

File: pubtools/_quay/operator_pusher.py [hunk 2]
```diff
@@ -513,10 +513,10 @@ def build_index_images(self):
...
                 if not items_opted_in[id(item)] or (
-                    items_opted_in[id(item)] and osev_tuple <= (4, 10)
+                    items_opted_in[id(item)] and osev_tuple <= (4, 12)
                 ):
                     non_fbc_items.append(item)
-                elif items_opted_in[id(item)] and osev_tuple >= (4, 11):
+                elif items_opted_in[id(item)] and osev_tuple >= (4, 13):
                     LOG.warning(
                         "Skipping {i}".format(i=item)
                         + "from iib build as it's opted in for FBC and targeting OCP version >4.10"
```
Comment (querti) [inline pubtools/_quay/operator_pusher.py:522]: This is incorrect, right?

**Review by querti — CHANGES_REQUESTED** [review]

**Review by midnightercz — COMMENTED** [review]
The Change looks good, but you need to fix log messages

File: pubtools/_quay/operator_pusher.py (hunk 2 above)
Comment (yashvardhannanavati) [reply pubtools/_quay/operator_pusher.py:522]: Nice catch! Changing it.

Comment (yashvardhannanavati) [reply pubtools/_quay/operator_pusher.py]: Done

**Review by chandwanitulsi — APPROVED** [review]

**Review by emilyzheng — APPROVED** [review]

**Review by querti — APPROVED** [review]

**Review by lipoja — APPROVED** [review]

**Pull request merged**